homeassistant/components/alexa/capabilities.py

```diff
@@ -169,6 +169,11 @@ def supported_operations():
         """Return the supportedOperations object."""
         return []
 
+    @staticmethod
+    def camera_stream_configurations():
+        """Applicable only to CameraStreamController."""
+        return []
```

Comment:
Should the default be `None` so we don't generate unused lists ?

Reply:
Yes.
If I remember correctly; some of those methods do need to return empty list/array for discovery.  But most of them should be set to `None`. I'll review each of the static methods and submit a new PR.